assets/model_monitoring/components/generation_safety_quality/annotation_compute_metrics/spec.yaml

```diff
@@ -60,23 +60,20 @@ conf:
   spark.synapse.library.python.env: |
     channels:
       - defaults
-      - conda-forge
     dependencies:
       - python=3.10
       - pip:
-          - azure-cli-core~=2.45.0
-          - azure-identity~=1.12.0
-          - azure-ai-ml~=1.10.0
-          - azure-keyvault-secrets==4.7.0
-          - tiktoken==0.3.3
-          - json5==0.9.11
-          - scipy~=1.10.0
-          - numpy~=1.21.0
-          - pandas~=1.4.3
-          - azureml-mlflow~=1.49.0
-          - azure-storage-file-datalake~=12.8.0
-          - mltable~=1.3.0
-
+        - azure-cli-core~=2.56.0
+        - azure-ai-ml~=1.13.0
+        - azure-ai-generative==1.0.0b7
+        - azure-ai-resources==1.0.0b7
+        - openai~=1.11.1
+        - json5==0.9.11
+        - mltable~=1.5.0
+        - promptflow[azure]~=1.4.1
+        - promptflow-tools~=1.1.0
+        - keyrings.alt~=5.0.0
+    name: momo-base-spark
```

Comment:
maybe the name here should be different since it's different from others - something like momo-gsq?

Reply:
I can update the name